packages/pigeon/lib/objc_generator.dart

```diff
@@ -747,7 +753,9 @@ void _writeFlutterApiSource(Indent indent, ObjcOptions options, Api api) {
     if (func.arguments.isEmpty) {
       sendArgument = 'nil';
     } else {
-      sendArgument = '@[${argNames.join(', ')}]';
+      String makeNullSafeExpression(String x) =>
```

Comment:
Nit: "null safe" is a pretty overloaded term. How about `cocoaCollectionSafeExpression`, since it seems like that's what this really means.

Reply:
switched to `makeVarOrNsnullExpression`

Reply:
`NSNull` is a two-letter abbreviation followed by a word, so `NSNull` would be the correct Dart style. (Unlike iOS, this is not ambiguous: class names from other languages are not single words, they are the result of applying that language's naming conventions to a series of words.)

Reply:
This wasn't resolved.

Reply:
Done